shared/domains/chelonia/chelonia.js

```diff
@@ -1198,6 +1200,58 @@ export default (sbp('sbp/selectors/register', {
     })
     return msg
   },
+  'chelonia/out/ownResources': async function (contractID: string) {
+    if (!contractID) {
+      throw new TypeError('A contract ID must be provided')
+    }
+
+    const response = await fetch(`${this.config.connectionURL}/ownResources`, {
+      method: 'GET',
+      signal: this.abortController.signal,
+      headers: new Headers([
+        [
+          'authorization',
+          buildShelterAuthorizationHeader.call(this, contractID)
+        ]
+      ])
+    })
+    if (!response.ok) {
+      throw new Error(`Unable to fetch own resources for ${contractID}`)
+    }
```

Comment:
Could you include a copy of the HTTP statuscode and status message in this error?